index.js

```diff
@@ -109,22 +109,25 @@ function buildAssets (opts, callback) {
             }
             rawScss.push('@import "'+opts.variables+'";');
         }
+        rawScss.push('@import "'+getModulePath('slate')+'/source/stylesheets/_variables.scss";');
         rawScss.push('@function font-url($url){ @return url($url) }');
-        rawScss.push('@media screen { @import "'+getModulePath('slate')+'/source/stylesheets/screen.css.scss"; }');
-        rawScss.push('@media screen { .highlight._{ @import "'+getModulePath('highlight.js')+'/../styles/solarized-light"; } }');
```

Comment:
The Slate scss is still being pulled in (see line 122 below) but only when a user hasn't specified their own scss file.
The highlight.js css is still being pulled in (see line 126 below). It wasn't working being imported and had to be referenced separately. I've now tested pretty thoroughly (hence the 3 commits to one pull request) do you think I've broken something?